.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,13 @@
 <!--- Make sure to add a descriptive title in the field above! E.g. "Fixes the header title color on the homepage"  -->
 
+## Ember Times review template
+
+- [ ] Author's name at top and bottom
+- [ ] Put emoji in writeup title
+- [ ] Link to external article/repo/etc in paragraph/body text, not just the header title (link in header title is optional now)
```

Comment:
Looking at lines 6 and 7, I think it'd be good to consistently use the word writeup (or header, if you prefer).

```suggestion
- [ ] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
```